desktop/app/mainWindow/index.js

```diff
@@ -59,8 +61,20 @@ function showAppWindow() {
 	mainWindow.webContents.loadURL( `file://${ getPath( 'index.html' ) }` );
 
 	mainWindow.setBrowserView( mainView );
-	mainView.setBounds( { ...bounds, ...{ x: 0, y: 38 } } );
-	mainView.setAutoResize( { horizontal: true, vertical: true } );
+	mainView.setBounds( {
+		x: 0,
+		y: TITLE_BAR_HEIGHT,
+		width: bounds.width,
+		height: bounds.height - TITLE_BAR_HEIGHT,
+	} );
+	mainWindow.on( 'will-resize', ( _, newBounds ) => {
+		mainView.setBounds( {
+			x: 0,
+			y: TITLE_BAR_HEIGHT,
+			width: newBounds.width,
+			height: newBounds.height - TITLE_BAR_HEIGHT,
+		} );
+	} );
```

Comment:
In testing on Windows this doesn't seem to be working well. I've been playing around trying to find a solution. This is pretty hacky but it seems to work well on both Mac and Windows, I replaced the `will-resize` with this.

```
mainWindow.on( 'resize', function () {
		const newBounds = mainWindow.getBounds();

		// Windows doesn't resize properly and requires extra space added to fit after resize.
		const boundsPadding =
			process.platform == 'win32'
				? { width: 20, height: TITLE_BAR_HEIGHT + 55 }
				: { width: 0, height: TITLE_BAR_HEIGHT };

		mainView.setBounds( {
			x: 0,
			y: TITLE_BAR_HEIGHT,
			width: newBounds.width - boundsPadding.width,
			height: newBounds.height - boundsPadding.height,
		} );
	} );
```

Reply:
I pushed a commit with this update in it.

Reply:
I tested the new change - using `resize` instead of `will-resize` and seems to work reasonably well on Windows, Mac and Linux. Nice catch, @sandymcfadden - if you could approve this PR will go ahead and get it landed.